### ISSUE TITLE
Missing images for yamllint after v1.26

### ISSUE THREAD

**Issue by amimas** [cytopia/docker-yamllint#29]
Hello, I couldn't find the images for yamllint after version 1.26. The latest tagged image does have the latest version of the yamllint though (i.e. 1.32.0), but would be great to have the images that are tagged with the version of the tool.

**Comment by meyfa** [comment]
I second this. It'd be great if cytopia/yamllint tags would correspond with yamllint versions. In other words, a new cytopia/yamllint tag should be created when updating yamllint. This can be automated via Dependabot by configuring it to look at the latest GitHub release for yamllint and create a PR bumping a variable in the Makefile.

This would greatly simplify getting reproducible lint results while staying up-to-date. Currently, we either have to pin the image SHA, in which case we can't really know what changed in the image, or use a point-in-time release such as `cytopia/yamllint:latest-0.14`, but then we don't get any yamllint updates.

I think issue #20 had a similar goal in mind.

By the way, thanks for putting in the work to provide this image in the first place, it's appreciated!

**Comment by lusu007** [comment]
Any news on this? 😊😅

**Comment by mjbear** [comment]
> I second this. It'd be great if cytopia/yamllint tags would correspond with yamllint versions. In other words, a new cytopia/yamllint tag should be created when updating yamllint. This can be automated via Dependabot by configuring it to look at the latest GitHub release for yamllint and create a PR bumping a variable in the Makefile.
> 
> This would greatly simplify getting reproducible lint results while staying up-to-date. Currently, we either have to pin the image SHA, in which case we can't really know what changed in the image, or use a point-in-time release such as `cytopia/yamllint:latest-0.14`, but then we don't get any yamllint updates.
> 
> I think issue #20 had a similar goal in mind.
> 
> By the way, thanks for putting in the work to provide this image in the first place, it's appreciated!

@cytopia
I agree with the others on this thread that matching the container image version to the version of yamllint would be helpful.

It would also minimize the information needed in those Markdown tables for [rolling and point in time releases](https://github.com/cytopia/docker-yamllint#whale-available-docker-image-versions).

It would be handy to have the Dockerhub images also have version tagging (or am I missing something)?
https://hub.docker.com/r/cytopia/yamllint/tags

Thoughts?
Thank you!
